users/arkag/arkag.c

```diff
@@ -0,0 +1,576 @@
+#include "arkag.h"
+#include "quantum.h"
```

Comment:
You shouldn't need all of these includes,  just the arkag.h, and the ones for the unicode stuff.  (eg, you don't need quantum, action, or the process_record)